src/main/resources/swagger.api/api-common.yaml

```diff
@@ -4,7 +4,7 @@ components:
       type: string
       format: uuid
     error:
-      $ref: schemas/error.json
+      $ref: schemas/error.json#/Error
```

Comment:
This change is to leave generated pojo as is (Error). Without this change there's invalid pojo generated for some reason..